showcase/tests/integration/components/hds/layout/grid/index-test.js

```diff
@@ -71,7 +91,7 @@ module('Integration | Component | hds/layout/grid/index', function (hooks) {
     assert.dom('#test-layout-grid').hasTagName('section');
   });
 
-  // ALIGN
+  // Align
```

Comment:
[nit] in the test files, we tend to use all uppercase to make them more visible when scrolling the code (they tend to be long files)